### DISCUSSION TITLE
Request optimization software: win10 can not be used, suspected that the motherboard does not support avx512

### DISCUSSION THREAD

**Discussion by zxmtangfan** [madMAx43v3r/chia-plotter#140] [Ideas]
为什么软件编译出来的X79主板的机器P不了，好像缺少AVX512

**Comment by zxmtangfan** [comment]
编译的WIN10用不了，卡住不动。

**Reply by zxmtangfan** [reply]
Request optimization software: win10 can not be used, suspected that the motherboard does not support avx512

**Comment by Hardcore-fs** [comment]
build round it......... old osx cpu also don't support AVX512
![bob](https://user-images.githubusercontent.com/1309004/121596975-b6fc4280-ca72-11eb-8034-ef4422866e7c.jpeg)

![d1](https://user-images.githubusercontent.com/1309004/121599177-43a80000-ca75-11eb-807e-fc6fa7846e89.jpeg)

**Comment by madMAx43v3r** [comment]
See: https://github.com/madMAx43v3r/chia-plotter/issues/126

Issue might be fixed in latest master

**Reply by zxmtangfan** [reply]
When will the new version come out？

**Reply by madMAx43v3r** [reply]
It seems the issue is not fixed yet
